tests/containers_test.go

```diff
@@ -399,6 +399,10 @@ func TestEnvironment(t *testing.T) {
 			if imageVariant == "pulumi" {
 				expectedPath += ":/root/.pulumi/bin"
 			}
+			// When running in bash, the UBI images include /root/.local/bin:/root/bin:
```

Comment:
Is this intentional, or de we need to exclude these?  Does this pull in any additional binaries for the user that might not be expected?